openshift/serverless.bash

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env bash
+
+function ensure_serverless_installed {
+  logger.info 'Check if Serverless is installed'
+  local prev=${1:-false}
+  if oc get knativeserving.operator.knative.dev knative-serving -n "${SERVING_NAMESPACE}" >/dev/null 2>&1 && \
+     oc get knativeeventing.operator.knative.dev knative-eventing -n "${EVENTING_NAMESPACE}" >/dev/null 2>&1
+  then
+    logger.success 'Serverless is already installed.'
+    return 0
+  fi
+
+  # Deploy config-logging configmap before running serving-opreator pod.
+  # Otherwise, we cannot change log level by configmap.
+  enable_debug_log
+
+  if [[ $prev == "true" ]]; then
+    install_serverless_previous
+  else
+    install_serverless_latest
+  fi
+}
+
+function install_serverless_previous {
+  local rootdir
+  rootdir="$(dirname "$(dirname "$(dirname "$(realpath "${BASH_SOURCE[0]}")")")")"
+
+  # Remove installplan from previous installations, leaving this would make the operator
+  # upgrade to the latest version immediately
+  remove_installplan "$CURRENT_CSV"
```

Comment:
Where is this defined? Is it read from the serverless-operator project somehow? I don't see this in knative-eventing project.

Reply:
its read from operator repo [here](https://github.com/openshift/knative-eventing/pull/765/files#diff-cd758740b9c6da1c23340da7084242feR71)
```
  git clone --branch release-1.8 https://github.com/openshift-knative/serverless-operator.git $operator_dir
  cp openshift/serverless.bash $operator_dir/hack/lib/serverless.bash
  pushd $operator_dir
  ./hack/install.sh && header "Serverless Operator installed successfully" || failed=1
```

Reply:
@mgencur  this is a "hack".  I have changed the `serverless.bash` file, that it does NOT install eventing - just serving

Perhaps the `install.sh`, could use arguments, like 

```sh
./hack/install.sh serving eventing
```

but my hack was quicker :man_shrugging:

Reply:
Actually, it looks like its defined in serverless-operator/hack/lib/vars.bash. The install.sh script "sources" that file no matter where it's called from